org.civicrm.reqangular/src/common/services/session.js

```diff
@@ -42,4 +42,4 @@
       };
     }]);
   });
-})(CRM, Drupal);
+})(CRM, window.Drupal);
```

Comment:
yes, but it's being used only at one place that is still not merged to la-milestone-3 so there's no worry (for this case)